string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,9 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  var newWord = word.toLowerCase();
+  newWord = newWord.slice(0, 1).toUpperCase() + newWord.slice(1);
+  if (newWord === 'Javascript') {
+    newWord = newWord.slice(0, 1).toUpperCase() + newWord.slice(1, 4) + newWord.slice(4, 5).toUpperCase() + newWord.slice(5);
```

Comment:
You can just `return "JavaScript"`, this is a very specific value, so you don't need to build it programmatically

Reply:
sounds good thanks! 
![image](https://user-images.githubusercontent.com/79817237/213814136-7bc175fe-367a-4070-8059-6a88e369ca91.png)